INSTALL.md

```diff
@@ -55,6 +55,29 @@ ruby -v
 
 > Mac OS X 10.13 comes with 2.3.7 but you might also use [rbenv](https://github.com/rbenv/rbenv) or [rvm](https://rvm.io/) to install your own ruby version.
 
+#### Node version
+
+The project supports **Version: 8.X.X**.
+
+You might want to use [nvm](https://github.com/creationix/nvm/) to install and work with a specific Node versions:
```

Comment:
Typo, it should be `specific Node version`

Reply:
I think I wanted `with specific node versions` without the `a` hehe, thanks!